controller/user.js

```diff
@@ -192,25 +56,34 @@ async function update(req, res, next) {
   });
 }
 
-async function remove(req, res) {
-  const id = getIdParam(req);
-  await models.user.destroy({
-    where: {
-      id: id,
-    },
-  });
-  res.status(statusCode.SUCCESS).end();
+async function changePassword(req, res, next) {	
+  const user_id = req.user.id;	
+  const user = await models.User.findByPk(user_id);	
+  if (!user) {	
+    throw new errors.FcError(errors.USER_NOT_FOUND);	
+  }	
+
+  if (!bcrypt.compareSync(req.body.old_password, user.password)) {	
+    throw new errors.FcError(errors.EMAIL_USERNAME_PASSWORD_INVALID);	
+  }	
+
+  const hashedPassword = getHashedPassword(req.body.new_password);	
+  user.password = hashedPassword;
+  user.save().then((updatedUser) => {
+    if (!updatedUser) {
+      throw new errors.FcError(errors.SYSTEM_ERROR);
+    }
```

Comment:
remove this unnecessary check

See https://sequelize.org/master/class/lib/model.js~Model.html#instance-method-save